src/wallet/rpcwallet.cpp

```diff
@@ -1332,7 +1332,7 @@ static void MaybePushAddress(UniValue & entry, const CTxDestination &dest)
 /**
  * List transactions based on the given criteria.
  *
- * @param  pwallet        The wallet.
+ * @param  wallet         The wallet.
```

Comment:
Good catch! 👍 `develop` is currently broken because of this https://gitlab.com/dashpay/dash/-/pipelines/1381488325